server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+const port = 3003;
+var db = require ('../database/index.js');
+const bodyParser = require('body-parser');
+
+app.use(express.static('./public'));
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(bodyParser.json());
+
+app.get('/:experience_id', (req, res) => {
+  db.getPhotos(req.params.experience_id, (error, exData) => {
+    if (error) {
+      res.status(400).send(error);
+    } else {
+      res.status(200).send(exData)
+    }
+  });
+});
+
+
+app.listen(port, () => console.log(`App listening on port ${port}`));
+
+console.log('testing');
```

Comment:
another console.log.  just out of curiosity, what are you using these for?

Reply:
Oops, they were supposed to be deleted! That was me trying to make changes so that my pull request would show all needed files, but it ended up being another issue. I think they came back when I reverted to a previous version, good catch, I will delete!